pkgs/by-name/co/codecrafters-cli/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  git,
+}:
+buildGoModule rec {
+  pname = "codecrafters-cli";
+  version = "29";
+
+  src = fetchFromGitHub {
+    owner = "codecrafters-io";
+    repo = "cli";
+    rev = "v${version}";
+    hash = "sha256-h8W9MHKnJCktVn8ZL3agSD7PcihF3Ow/KEElSIPUPlM=";
+  };
+
+  vendorHash = "sha256-TQcxzfiqKeCQZUKLHnPjBa/0WsYJhER3fmr4cRGFknw=";
+
+  nativeBuildInputs = [git];
```

Comment:
Looks like it expects `git` from `PATH`, try to `wrapProgram` it (in `makeWrapper`) in `postFixup`. `git`seems not needed in `nativeBuildInputs`.